### ISSUE TITLE
Doctrine Error When Loading Relationships.

### ISSUE THREAD

**Issue by devingray** [beyondcode/invoker-community#214]
I have a fresh install of Laravel 8 using PHP 8.0.3. I open my USER Tab and add a panel to load notifications relationship

<img width="1029" alt="CleanShot 2021-06-16 at 12 57 55@2x" src="https://user-images.githubusercontent.com/11573528/122165497-7e110300-cea2-11eb-9ddb-0f560240bc82.png">

This then throws the error below

Mysql Version is 8.0.19 and Also breaks on SQLITE

------
**OS**: darwin
**Invoker Version**: 2.5.1
**Laravel Version**: 8.47.0
**Local project**: true
**PHP Binary**: /usr/local/bin/php
**Route**: /project/show?id=1&class=App%5CModels%5CUser&project=local-jw4hKYqAPQL5sTnsMELwN9

**Error**:
```
Error 

  Class "Doctrine\DBAL\Driver\AbstractMySQLDriver" not found

  at /Users/devingray/CodeProjects/lemonswan3/vendor/laravel/framework/src/Illuminate/Database/PDO/MySqlDriver.php:8
      4▕ 
      5▕ use Doctrine\DBAL\Driver\AbstractMySQLDriver;
      6▕ use Illuminate\Database\PDO\Concerns\ConnectsToDatabase;
      7▕ 
  ➜   8▕ class MySqlDriver extends AbstractMySQLDriver
      9▕ {
     10▕     use ConnectsToDatabase;
     11▕ }
     12▕

      [2m+5 vendor frames [22m
  6   phar:/Applications/Invoker.app/Contents/Resources/invoker.phar/src/Factories/AttributeCollection.php:25
      Illuminate\Database\Connection::getDoctrineSchemaManager()

  7   phar:/Applications/Invoker.app/Contents/Resources/invoker.phar/src/Factories/AttributeCollection.php:15
      _HumbugBox548199e96c64\Invoker\Factories\AttributeCollection::getColumns("Illuminate\Notifications\DatabaseNotification")
```

**Comment by bkilshaw** [comment]
Running into the same issue using PG11. Requiring doctrine fixes it, but not an ideal solution.

`composer require doctrine/dbal`

**Comment by mpociot** [comment]
This is now fixed in Invoker 2.5.2
